tests/test_client.py

```diff
@@ -255,7 +265,14 @@ def test_xarray_converter(wps):
     assert isinstance(ncdata, xr.Dataset)
 
 
-def test_sort_inputs():
+def test_sort_inputs(process):
+    # The first three inputs are all minOccurs=1 with no default, so we expect them to remain in the same order.
+    ps = sorted(process.dataInputs, key=sort_inputs_key)
+    for i in range(3):
+        assert ps[i] == process.dataInputs[i]
```

Comment:
I think this willl not work in python 3. Maybe this:
```python
    ps = sorted(process.dataInputs, key=sort_inputs_key)
    for i1, i2 in zip(ps, process.dataInputs):
        assert i1 == i2
```

Reply:
Seems to be working with 3.6.3 at the very least.

Reply:
wow sorry, I was sure `sorted` returned a generator in python 3...